File: 0-1000 C/HELIUM3.c
#include <stdio.h>
int main(){
    int T, X, Y, A, B, pow_req, pow;
    scanf("%d", &T);
    for (int i = 0; i < T; i++){
        scanf("%d %d %d %d", &A, &B, &X, &Y);
        pow_req = A*B;
        pow = X*Y;

        (pow >= pow_req) ? printf("Yes\n") : printf("No\n");
     }
    return 0;
}

File: 0-1000/HELIUM3.py
T = int(input())

for i in range(T):
    temp = input().split(" ")
    A, B, X, Y = int(temp[0]), int(temp[1]), int(temp[2]), int(temp[3])

    pow_req = A*B
    power = X*Y

    if power >= pow_req:
        print("Yes")
    else:
        print("No")
